### PR TITLE
OTEL Install Collector for Ansible, Chef, Puppet

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Add instructions for installing OTEL collector for Chef, Puppet, Ansible. Do not merge until @ajaiswals approves.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205594552764910

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/install-collector/linux.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 ---
-id: install-collector-linux
-title: Installing Collector on Linux
-sidebar_label: Install Collector on Linux
-description: Learn how to install the Sumo Logic OpenTelemetry Collector.
+id: linux
+title: InstallOpenTelemetry Collector on Linux
```
Comment (jpipkin1) [inline docs/send-data/opentelemetry-collector/install-collector/linux.md:3]:
```suggestion
title: Install OpenTelemetry Collector on Linux
```

**Review by jpipkin1 — APPROVED** [review]
Looks great, @kimsauce. I just made one tiny tweak.

**Comment by kimsauce** [comment]
> Looks great, @kimsauce. I just made one tiny tweak.

Thanks, John! @ajaiswals , can you please review?

**Review by ajaiswals — APPROVED** [review]
Looks good. Thanks @kimsauce @jpipkin1

**Pull request merged**